src/main-process/start.js

```diff
@@ -79,7 +79,7 @@ module.exports = function start(resourcePath, devResourcePath, startTime) {
   }
 
   const releaseChannel = getReleaseChannel(app.getVersion());
-  let appUserModelId = 'com.squirrel.atom.' + process.arch;
+  let appUserModelId = 'dev.pulsaredit.pulsar.' + process.arch;
```

Comment:
```suggestion
  let appUserModelId = 'dev.pulsar-edit.pulsar.' + process.arch;
```

To match the domain `pulsar-edit.dev` (add missing hyphen)

Reply:
Good call, was going with the previous style, but does make more sense to have it properly match our domain.